entity-framework/core/providers/sqlite/built-in-functions.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Built-in Functions - SQLite Database Provider - EF Core
```

Comment:
Maybe "Function translations" or even just "Translations"? Specifically for Sqlite not everything here is "built-in" (i.e. all the `ef_` stuff).

I'd also link to the spatial page from here when that arrives (this way this page covers all translations), and that's not built-in either.

Reply:
I had that, but changed to match Smit’s docs. Happy to change back.
Definitely needs a link to spatial.

Reply:
Built-in functions is right one. Most of them are truly C# functions in LINQ side. Plus, I am gonna add text at the start that definition is bit blurry. The built-in part in this page is that EF Core provides this function translations rather than use defining it. (Comes in the box)